src/views/App/index.tsx

```diff
@@ -104,14 +104,8 @@ const AppRoutes = () => {
       {/* Model Routes */}
       <SecureRoute path="/models" exact component={ModelPlan} />
 
-      <Redirect
-        exact
-        from="/models/:modelID/read-only"
-        to="/models/:modelID/read-only/model-basics"
-      />
-
       <SecureRoute
-        path="/models/:modelID/read-only/:subinfo"
+        path="/models/:modelID/read-only/:subinfo?"
```

Comment:
What does the addition of `?` do to this route?

Reply:
The `?` makes the parameter optional.  The component can still be rendered without passing any of the subinfo (`model-basics`, `payments`, etc)

Reply:
test